src/cpu/dynarmic_wrapper/build.rs

```diff
@@ -57,6 +58,14 @@ fn main() {
         }
         build.define("Boost_INCLUDE_DIR", boost_path);
     }
+    let os = env::var("CARGO_CFG_TARGET_OS").expect("CARGO_CFG_TARGET_OS was not set");
+    if cfg!(target_os = "macos") && os.eq_ignore_ascii_case("android") {
+        // TODO: how to set version independent path?
```

Comment:
There's already a special solution for this on Windows, maybe you can just re-use this for Android and have the user set up a symlink? It's ugly of course, but it avoids hardcoding the path.

Reply:
Done

Reply:
Hmm this condition is a bit weird, shouldn't it be `os.eq_ignore_ascii_case("windows") || os.eq_ignore_ascii_case("android")`?

Reply:
😅 you're right, done